### PR TITLE
Allow easier extension of core classes

### PR DESCRIPTION
**Pull request by craig410**
So that you can extend EmailVerificationRequest to provide additional request types

### CODE REVIEW

**Review by acoulton — APPROVED** [review]
LGTM

File: src/Interactor/LoginInteractor.php [hunk 1]
```diff
@@ -118,7 +118,12 @@ protected function handlePasswordIncorrect(User $user)
             );
         }
 
-        return LoginResponse::passwordIncorrect($user);
+        return $user->isActive() ? LoginResponse::passwordIncorrect($user) : LoginResponse::notActive($user);
```
Comment (acoulton) [inline src/Interactor/LoginInteractor.php:121]: I guess for completeness should mention this response change in the changelog too.

**Review by acoulton — APPROVED** [review]
Fab - tiny tweak to changelog but the code/tests are all good.

File: src/Interactor/LoginInteractor.php (hunk 1 above)
Comment (craig410) [reply src/Interactor/LoginInteractor.php:121]: Done

**Pull request merged**